www/src/main.css

```diff
@@ -124,3 +124,23 @@ h6 {
     text-decoration: underline;
   }
 }
+
+
+/* Styling buttons */
+
+kbd {
```

Comment:
might be good to put these styles on a kbd that has a particular classname? 

then you can revive the `KeyboardButton` component and just make sure it uses `<kbd>` under the hood to remain semantically apropos

Reply:
targeting `kbd` directly has the side effect of styling, for example, the keyboard shortcuts for search (i linked to an image in my comment on this pr)

Reply:
ahhh yea good catch